src/osvr/Macros.cmake

```diff
@@ -34,10 +39,19 @@ macro(osvr_add_library)
 
     set_target_properties(${LIBNAME_FULL} PROPERTIES
         FOLDER "${LIB_FOLDER}")
-
-    generate_export_header(${LIBNAME_FULL}
-        BASE_NAME ${EXPORT_BASENAME}
-        EXPORT_FILE_NAME Export.h)
+        
+    # Are we building or using this library?
+    get_target_property(EXPORT_IMPORT_CONDITION ${LIBNAME_FULL} DEFINE_SYMBOL)   
```

Comment:
yeah, between 42 and 50 doesn't appear to actually even be used at all, and can probably be removed.